### ISSUE TITLE
create data loader and export for TensorFlow Protobuf files

### ISSUE THREAD

**Issue by thinkh** [Caleydo/malevo#1]
PyTorch code for exporting objectives

* accuracy
* loss
* classification

Result: matrix csv or HDF file for every epoch

**Comment by gfrogat** [comment]
## update of this issue:
due to the switch to TensorFlow we need to handle Protobuf model files now, and have to be able to export the relevant data into a format Phovea can handle.

## Requirements:
- CSV File
	- image id
	- ground truth label
	- predicted label
	- epoch id
	- softmax vector

- list of image ids for each pair (ground_truth, predicted)

**Comment by thinkh** [comment]
We will do this later, because it's too much engineering work for now and it works fine with CSV files.

**Comment by gfrogat** [comment]
Will also look into this over the next weeks, related to #33

**Comment by gfrogat** [comment]
The new data loader won't use protobuf as Tensorflow's Summary doesn't really fit our requirements well. We could export each sample as Protobuf tensor summary, but then we still need to load all the messages into a database.

I'll focus on a simple CSV export for now. CSV is framework agnostic and every common database already supports import of CSV files which simplifies the database setup a lot.
